sebs/aws/aws.py

```
@@ -430,47 +590,59 @@ def default_function_name(
 
-    def delete_function(self, func_name: Optional[str]):
+        Args:
+            func_name: Name of the function to delete
+
+        Note:
+            FIXME: does not clean the cache in SeBS.
+        """
         self.logging.debug("Deleting function {}".format(func_name))
         try:
             self.client.delete_function(FunctionName=func_name)
         except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)